### PR TITLE
AWS Guard Duty sets context on null value for aws-gd-get-members

### PR DESCRIPTION
**Pull request by daryakoval**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/37664

## Description
**aws-gd-get-members** returned null to context, when there is no members.

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty.py [hunk 1]
```diff
@@ -720,8 +714,12 @@ def get_members(args):
             AccountIds=accountId_list
         )
 
-        ec = {"AWS.GuardDuty.Members(val.AccountId === obj.AccountId)": response['Members']}
-        return create_entry('AWS GuardDuty Members', response['Members'], ec)
+        members_response = response.get('Members')
```
Comment (dantavori) [inline Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty.py:717]:
```suggestion
        members_response = response.get('Members', [])
```

File: Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty_test.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+
+class Client:
+    @staticmethod
+    def get_members(DetectorId, AccountIds):
+        return RESPONSE
+
+
+def test_get_members(mocker):
```
Comment (dantavori) [inline Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty_test.py:21]: please add a docstring (given/when/then) to define what are you testing (the bug fix)

File: Packs/AWS-GuardDuty/ReleaseNotes/1_1_3.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### AWS - GuardDuty
+- Fixed an issue where **aws-gd-get-members** returned null to context, when there is no members.
```
Comment (dantavori) [inline Packs/AWS-GuardDuty/ReleaseNotes/1_1_3.md:4]:
```suggestion
- Fixed an issue where ***aws-gd-get-members*** command returned a null value to the context when there were no members in the response.
```

**Review by dantavori — CHANGES_REQUESTED** [review]
niiice, see my minor notes

File: Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty_test.py (hunk 2 above)
Comment (daryakoval) [reply Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty_test.py:21]: done

**Review by dantavori — APPROVED** [review]

**Pull request merged**